lib/improver/tests/psychrometric_calculations/psychrometric_calculations/test_WetBulbTemperatureIntegral.py

```diff
@@ -60,24 +64,33 @@ def test_basic(self):
         self.assertEqual(result, msg)
 
 
-class Test_process(IrisTest):
+class Test_process(Test_WetBulbTemperature):
 
     """Test the calculation of the wet bulb temperature integral from
     temperature, pressure, and relative humidity information using the
     process function. Integration is calculated in the vertical."""
 
     def setUp(self):
         """Set up cubes."""
```

Comment:
I might be useful to expand this docstring slightly.

Reply:
Expanded a bit to say more about why we calculate such integrals. Is that what you were after?

Reply:
Oh, I meant the set up cubes one, not the one above.

Reply:
Oooops 😳